src/openMVG/sfm/sfm_data_io_gt.hpp

```diff
@@ -119,35 +126,47 @@ static bool read_Strecha_Camera(const std::string & camName, Pinhole_Intrinsic &
 
 /**
 @brief Reads a set of Pinhole Cameras and its poses from a ground truth dataset.
-@param[in] functorPointer, to the function which can handle the trajectory format. Example: &read_openMVG_Camera
 @param[in] sGTPath, the directory where the camera files are located.
```

Comment:
I don't know for this one. There is multiple solutions possible: 
1. Give the path of gt folder and consider there is at the same level a folder `images` (actual solution)
2. Give the path of a folder containing two folders `images` and `gt_dense_cameras`
3. Give the path of a folder containing two folders image and gt whatever their name, and use `stlplus::folder_wildcard` with recursive flag on to get `*.camera` files list first then `*.<imagetype>` list.

In two words, I hesitate between a moderate/high strict naming convention (**1** and **2**) and a more free mode (**3**) where the only restriction is to have (recursively) only 1 GT set and 1 image set in the root folder.